Contributing.md

```diff
@@ -56,6 +56,28 @@ discuss pending contributions in order to find a resolution. It is expected that
 small minority of issues be brought to the TC for resolution and that discussion and
 compromise among committers be the default resolution mechanism.
 
+# Triaging
+
+Open issues in this repo are subjected to the triage criteria:
+
+`needs triage`: Issues that are opened and un-attended / unclassified belong to this 
+category, and are either automatically or manually labelled with `needs triage`
+
+`question`: Issues that are not leading upto a bug or an enhancement to any of the 
+express repos, but emanating from the usage of it are classified as `question`
+
+`bug`: Issues that, after investigation, provides a reasonable conviction about
+a potential bug in the express code, are classified as `bug`
+
+`enhancement`: Issues that, after discussion, inventigation, and technical assessment,
```

Comment:
I think this is a typo.

inventigation -> investigation

Reply:
@dougwilson - done, thanks; PTAL!